cmd/root.go

```diff
@@ -535,10 +706,18 @@ func updateComment(ghComment github.IssueComment, jComment jira.Comment, jIssue
 		return nil
 	}
 
-	user, _, err := ghClient.Users.Get(context.Background(), *ghComment.User.Login)
+	u, _, err := makeGHRequest(func() (interface{}, *github.Response, error) {
+		return ghClient.Users.Get(context.Background(), *ghComment.User.Login)
+	})
 	if err != nil {
 		log.Errorf("Error retrieving GitHub user %s. Error: %v", *ghComment.User.Login, err)
 	}
+	user, ok := u.(*github.User)
+	if !ok {
+		log.Errorf("Get GitHub user did not return user! Got: %v", u)
+		return errors.New(fmt.Sprintf("Get GitHub user failed: expected *github.User; got %T", u))
```

Comment:
Errors should not start with a capital because it is convention to wrap errors in more errors and this results in capitalized words in the middle of sentences. if you use golint, these mistakes will be caught.